packages/pyxis-react/src/stories/components/Button/Overview.tsx

```diff
@@ -26,17 +27,39 @@ const overviewDescription = (
     <p>
       All the properties described below concern the visual implementation of the component,
       in addition to these properties the component also supports all the&nbsp;
-      <a className="link" href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2076" rel="noreferrer" target="_blank">properties</a>
-      &nbsp;of a&nbsp;
+      <a
+        className="link"
+        href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2059"
```

Comment:
Come mai questi link sono cambiati?
Dovrebbe essere:
```suggestion
       href="https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2075"
```